self-hosted-runner.tf

```diff
@@ -0,0 +1,125 @@
+provider "aws" {
+  region = "eu-west-1"
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+}
+
+variable "aws_access_key" {
+  description = "AWS Access Key"
+  type        = string
+}
+
+variable "aws_secret_key" {
+  description = "AWS Secret Key"
+  type        = string
+}
+
+variable "aws_key_name" {
+  description = "AWS Key Name"
+  type = string
+}
+
+resource "aws_instance" "self_hosted_runner" {
+  ami           = "ami-055032149717ffb30" # change to ami-0776c814353b4814d when creating an AMI. 
+  instance_type = "t2.xlarge"
+
+  root_block_device {
+    volume_size = 16  // GB
+  }
+
+  key_name = var.aws_key_name
+  
+  tags = {
+    Name = "kuadrant-operator-self-hosted-runner"
+  }
+
+  // Security Group for SSH, HTTP, and HTTPS access
+  security_groups = ["ssh-http-https-access"]
+
+  # Uncomment when creating an AMI . 
```

Comment:
When should a new AMI be created?

Reply:
The new AMI should be created when dependencies are updated, such as kind, kubernetes, and podman. :)